crawl-ref/source/webserver/game_data/static/action_panel.js

```diff
@@ -451,7 +507,24 @@ function ($, comm, client, cr, enums, options, player, icons, gui, main,
         // TODO: select tile via something like c++ `tileidx_command`
         draw_action(gui, gui.PROMPT_NO, null, 0, adjusted_scale, selected == 0);
         draw_action(gui, gui.CMD_GAME_MENU, null, inc, adjusted_scale,
-                    selected == 1);
+            selected == 1);
```

Comment:
many (or maybe all) of the relevant commands have icons already for local tiles buttons, it'd be better to use those. Some of these will be fairly confusing in this use, e.g. the airstrike icon doesn't cast airstrike, etc.